src/main.cpp

```diff
@@ -47,15 +55,31 @@ static bool _get_options(int argc, char **argv, string &stream, string &hwid, st
 	desc.add_options()
 		("help", "print usage")
 
-		("hwid,i", po::value<string>(&hwid),
-		 "The hardware-id of the device. If not provided the script will look for"
-		 "the current ostree sha in the tufrepo and find the hardware-id from that.")
-
 		("stream,s", po::value<string>(&stream)->default_value(streams[0]),
 		 "The update stream to subscribe to: " DEVICE_STREAMS)
 
+		("hwid,i", po::value<string>(&hwid),
+		 "An identifier for the device's hardware type. If not provided, "
+                 "the current OSTree SHA in the TUF repo will be used to determine "
+		 "a value.")
```

Comment:
is this a tabs/space thing or something? The alignment of the middle line in description is off by one column.

Reply:
Yep, nice catch, thanks!